cypress/support/api/data-import.js

```diff
@@ -0,0 +1,78 @@
+Cypress.Commands.add('createMappingProfileApi', (mappingProfile) => {
+  return cy.okapiRequest({
+    method: 'POST',
+    path: 'data-import-profiles/mappingProfiles',
+    body: {
+      profile: { ...mappingProfile }
+    },
+  });
+});
+
+Cypress.Commands.add('createActionProfileApi', (actionProfile) => {
+  return cy.okapiRequest({
+    method: 'POST',
+    path: 'data-import-profiles/actionProfiles',
+    body: {
+      ...actionProfile
+    },
+  });
+});
+
+Cypress.Commands.add('createJobProfileApi', (jobProfile) => {
+  return cy.okapiRequest({
+    method: 'POST',
+    path: 'data-import-profiles/jobProfiles',
+    body: {
+      ...jobProfile
+    },
+  });
+});
+
+Cypress.Commands.add('getId', () => {
+  return cy.okapiRequest({
+    method: 'GET',
+    path: 'data-import/uploadDefinitions?query=(status%3D%3D(%22NEW%22%20OR%20%22IN_PROGRESS%22%20OR%20%22LOADED%22))%20sortBy%20createdDate%2Fsort.descending&limit=1'
+  });
```

Comment:
```suggestion 
  return cy.okapiRequest({
      path: 'data-import/uploadDefinitions',
      searchParams: {
        query: '(status==("NEW" OR "IN_PROGRESS" OR "LOADED")) sortBy createdDate/sort.descending',
        limit: 1
      },
    });
```

Reply:
@TetianaParanich , let's not hard code url params and please clean up generated files in `fixtures` and `downloads`

Reply:
thanks, I got it. I changed it